main.js

```diff
@@ -44,45 +39,4 @@ function updatePalette(array) {
 		allBoxes[i].style.backgroundColor = array[i];
 		allHexes[i].innerText = array[i];
 	}
-}
-
-function changeLock() {
```

Comment:
Functionality intact. This function allowed us to complete iteration 2. Great work!